banyan/Cargo.toml

```diff
@@ -21,7 +21,7 @@ parking_lot = "0.11.1"
 smallvec = "1.6.1"
 tracing = "0.1.25"
 weight-cache = "0.2.3"
-zstd = "0.8.0"
```

Comment:
Had to revert the update - since I had some weird perf problems with 0.8.0, and I don't want to look into those right now.